docs/feedback.txt

```diff
@@ -0,0 +1,36 @@
+
+Very helpful docs! Especially the stuff on the low level design and rationale
+for 'object oriented' style c. Notes below from python 3.6 on ubuntu 16.10
+using commit 36779a8.
+
+1. the python install instructions worked fine for me (following the
+instructions to install sys reqs, using `pip install .` and `pip install -r
+requirements.txt`)
+
+2. nosetests gave some failures without numpy and h5py installed (noted in PR)
+
+3. running `make` in the `lib` directory worked to build the c cli
+
+4. after install, running `msp` or `mspms` from the command line gave me an
+error `pkg_resources.DistributionNotFound: The 'svgwrite' distribution was not
+found and is required by msprime` is there some additional sysreq for svgwrite?
+It appears with `pip freeze` and I can load it fine in python.
```

Comment:
This was apparently due to a system-level install of `msprime` interfering with my version (installed locally using `pip -e .`) when I build and run locally, as now explicitly suggested, things work fine